client/client.go

```diff
@@ -497,6 +523,9 @@ func (h *HttpsClient) Validate() {
 		}
 		if h.Key == "" {
 			log.Error("The 'Certificate' field is set in the mTLS configuration, but no 'Key' is given. Both need to be present in order for mTLS to function")
+		} else if strings.HasPrefix(h.Key, pkcs11URIPrefix) && len(h.SSLEngine) < 1 {
```

Comment:
```suggestion
		} else if strings.HasPrefix(h.Key, pkcs11URIPrefix) && len(h.SSLEngine) == 0 {
```

Reply:
I guess this could also be `h.SSLEngine == ""` ?

Reply:
sure. changed to the suggestion. (I usually use len()<1 somehow I feel safer ;-))